applications/sdp/mspi/src/hrt/hrt.h

```diff
@@ -70,6 +70,10 @@ typedef struct {
 
 	/** @brief Function for writing to buffered out register. */
 	void (*vio_out_set)(uint32_t value);
+
+	/** @brief Function for reading from buffered in register. */
+	uint32_t (*vio_inb_get)(void);
+
```

Comment:
Why is it passed as a pointer? I see only one function used, so could be hardcoded.